### PR TITLE
Switch to tarball and skip compression

### PR DESCRIPTION
**Pull request by chuan-wang**

### CODE REVIEW

**Comment by chuan-wang** [comment]
Associated with validation 24_15. /CW

**Review by ssjunnebo — APPROVED** [review]
Looks good 👍

File: taca/backup/backup.py
Comment (alneberg) [inline taca/backup/backup.py:303]: Did you mean to remove one level of indentation here? That changes the logic of the if/elif statements. I'm not sure which one is correct.

Comment (chuan-wang) [reply taca/backup/backup.py]: > Did you mean to remove one level of indentation here? That changes the logic of the if/elif statements. I'm not sure which one is correct.

Good catch! Thanks! It has been fixed

**Review by alneberg — APPROVED** [review]
Very nice!

**Comment by chuan-wang** [comment]
All tests in validation 24_15 passed. It is waiting for approval.

**Review by parlundin — APPROVED** [review]

**Comment by chuan-wang** [comment]
PR merged after approval by PLu and JoA. The validation 24_15 is waiting for the QA group to finalize. /CW

**Pull request merged**
